src/loaders/express.ts

```diff
@@ -0,0 +1,16 @@
+import express, {Response, NextFunction} from 'express';
+import {RESPONSE_MESSAGES} from '../config/messages';
+import usersRouter from '../routers/users-routes';
+
+export default ({app}: {app: express.Application}) => {
+  app.use(express.json());
+
+  app.use('/users', usersRouter);
+
+  app.use((err: unknown, res: Response, next: NextFunction) => {
```

Comment:
it doesn't work.
1. It has to be defined before router
2. You are missing req param. Please, take a look at docs

Reply:
2. added missing req param.
1. seems like router must be defined before, otherwise it doesn't work (on my side)